public/static/js/task.js

```diff
@@ -267,6 +267,25 @@ mciModule.controller('TaskHistoryDrawerCtrl', function($scope, $window, $locatio
         $scope.taskHost = $window.taskHost;
         $scope.jiraHost = $window.jiraHost;
         $scope.subscriptions = [];
+
+        $scope.validateDuration = function(duration) {
+          if (!Number.isInteger(+duration)) {
+            return duration + " must be an integer";
+          }
+          if (+duration < 0) {
+            return duration + " cannot be negative";
+          }
+          return "";
+        }
+        $scope.validatePercentage = function(percent) {
+          if (isNaN(percent)) {
```

Comment:
Did you mean to  add `percent = parseFloat(percent)` at the top of this function? Additionality, Infinity and -Infinity

Reply:
Added checks for infinity